src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -911,6 +931,10 @@ def get_devices(_sys_block_path='/sys/block', device=''):
         metadata['sectorsize'] = get_file_contents(sysdir +
                                                    "/queue/logical_block_size",
                                                    fallback_sectorsize)
+        fallback_rotational = '1'
+        metadata['rotational'] = get_file_contents(sysdir +
+                                                   "/queue/rotational",
+                                                   fallback_rotational)
```

Comment:
How does this relate to the above `is_rotational`?

Reply:
There are already some fallbacks in place (i.e. sector size) so I added this one as well. This *should* not be needed, but does not hurt.

Reply:
I think Anthony's question was more about the fact that `is_rotational()` and `metadata['rotational']` don't seem to be related to each other. Maybe they should be?

Reply:
Indeed, I discovered the multiple implementations with our RGW Lua work a while back.  I can see where multiple ways of determining HDD vs SSD might return different results, leading to even more confusion.